ws-client/src/transport.rs

```diff
@@ -63,6 +63,8 @@ pub struct WsTransportClientBuilder<'a> {
 	/// `Origin` header to pass during the HTTP handshake. If `None`, no
 	/// `Origin` header is passed.
 	origin: Option<Cow<'a, str>>,
+	/// Max payload size
+	max_payload_size: usize,
```

Comment:
I think this is a better name and it'd be good to be consistent, so s/max_request_body_size/max_payload_size/ perhaps?